app/forms/country_form.rb

```diff
@@ -1,29 +1,25 @@
 class CountryForm
   include ActiveModel::Model
 
-  attr_accessor :country_code, :eligibility_check
+  attr_accessor :location, :eligibility_check
 
-  validates :country_code, presence: true
+  validates :location, presence: true
   validates :eligibility_check, presence: true
 
   def save
     return false unless valid?
 
-    eligibility_check.country_code = country_code
+    eligibility_check.country_code = location.split(":")[1]
     eligibility_check.save!
   end
 
-  def eligible?
-    eligibility_check.country_code != "INELIGIBLE"
-  end
-
   def success_url
-    unless eligible?
-      return(
-        Rails.application.routes.url_helpers.teacher_interface_ineligible_path
-      )
+    if eligibility_check.eligible_country_code?
+      Rails.application.routes.url_helpers.teacher_interface_degree_path
+    elsif eligibility_check.legacy_country_code?
+      "https://teacherservices.education.gov.uk/MutualRecognition/"
```

Comment:
Ah, this explains the `allow_other_host` from earlier.

We can make this an intermediary page in a follow-up PR once we have a design for it, and then it can be a regular `link_to` in the page

Reply:
Yep, agreed.